lib/Core/Site/Values/Content.php

```diff
@@ -283,4 +307,44 @@ private function getContentInfo()
 
         return $this->contentInfo;
     }
+
+    /**
+     * @return \Netgen\EzPlatformSiteApi\API\Values\Content
+     */
+    private function getOwner()
+    {
+        if ($this->owner === null) {
+            try {
+                $this->owner = $this->site->getLoadService()->loadContent($this->contentInfo->ownerId);
+            } catch (NotFoundException $e) {
+                $this->owner = false;
+            }
+        }
+
+        if ($this->owner === false) {
+            return null;
+        }
+
+        return $this->owner;
+    }
+
+    /**
+     * @return \eZ\Publish\API\Repository\Values\User\User
+     */
+    private function getInnerOwnerUser()
+    {
+        if ($this->innerOwnerUser === null) {
+            try {
+                $this->innerOwnerUser = $this->userService->loadUser($this->contentInfo->ownerId);
+            } catch (NotFoundException $e) {
+                $this->innerOwnerUser = false;
```

Comment:
Same remark as above. 
And you could get rid of the condition block below

Reply:
Updated in https://github.com/netgen/ezplatform-site-api/pull/51/commits/03952bc210937e9abc4649facec317120581b516.